nav2_bringup/params/turtlebot3_waffle_bridge.yaml

```diff
@@ -0,0 +1,60 @@
+# replace clock_bridge
+- ros_topic_name: "clock"
+  gz_topic_name: "/clock"
+  ros_type_name: "rosgraph_msgs/msg/Clock"
+  gz_type_name: "gz.msgs.Clock"
+  direction: GZ_TO_ROS
+
+# no equivalent in TB3 - add
+- ros_topic_name: "joint_states"
+  gz_topic_name: "joint_states"
+  ros_type_name: "sensor_msgs/msg/JointState"
+  gz_type_name: "gz.msgs.Model"
+  direction: GZ_TO_ROS
+
+# replace odom_bridge - check gz topic name
+# gz topic published by DiffDrive plugin
+- ros_topic_name: "odom"
+  gz_topic_name: "/odom"
+  ros_type_name: "nav_msgs/msg/Odometry"
+  gz_type_name: "gz.msgs.Odometry"
+  direction: GZ_TO_ROS
+
+# replace odom_tf_bridge - check gz and ros topic names
+# gz topic published by DiffDrive plugin
+# prefix ros2 topic with gz
+- ros_topic_name: "tf"
+  gz_topic_name: "/tf"
+  ros_type_name: "tf2_msgs/msg/TFMessage"
+  gz_type_name: "gz.msgs.Pose_V"
+  direction: GZ_TO_ROS
+
+# no equivalent in TB3 - add
+# gz topic published by PosePublisher
+# prefix ros2 topic with gz
+- ros_topic_name: "tf_static"
```

Comment:
Isn't the robot state publisher providing this? Why is the simulator publishing anything?

Reply:
Removed in https://github.com/ros-planning/navigation2/pull/3634/commits/6a462beb7b5d51992a66490739c10eb2fc3f2134